src/Input/Input.jsx

```diff
@@ -222,6 +237,7 @@ Input.propTypes = {
   icRight: PropTypes.string,
   id: PropTypes.oneOfType([PropTypes.string, PropTypes.number]).isRequired,
   label: PropTypes.node,
+  notRequiredText: PropTypes.node,
```

Comment:
Can you add a default prop value for this to be `label.optional`. Otherwise we would need to add this for every optional input in 3 portals.

Reply:
Done